ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift

```diff
@@ -46,25 +47,28 @@ class ButtonDemoController: UITableViewController {
         case .textAndIcon,
              .textOnly,
              .iconOnly:
-            let cell = UITableViewCell()
+            let cell = tableView.dequeueReusableCell(withIdentifier: ButtonDemoController.cellReuseIdentifier, for: indexPath)
```

Comment:
Should the name be more specific than just "cell", since you're dequeueing with a specific identifier?

Reply:
Ehhhhhh I think it's okay, because (A) it's still just a `UITableViewCell` being dequeued, and (B) there are no other cells in this context, so there isn't really any room for ambiguity.